MapleServer2/Managers/TrophyManager.cs

```diff
@@ -83,6 +83,24 @@ public static void OnTrigger(Player player, string trigger)
         UpdateMatchingTrophies(player, triggerTrophies, 1);
     }
 
+    public static void OnFall(Player player, float fallDistance)
+    {
+        // 23100013 "Living off the Edge", "Fall 9m X times"
+        IEnumerable<TrophyMetadata> fallSurviveTrophies = GetRelevantTrophies(TrophyTypes.FallSurvive);
+
+        // 22100005 "Icarus", "Fall X meter"
+        IEnumerable<TrophyMetadata> fallTrophies = GetRelevantTrophies(TrophyTypes.Fall);
+
+        if (fallDistance >= 900.0f)
+        {
+            UpdateMatchingTrophies(player, fallSurviveTrophies, 1);
```

Comment:
If you only need `fallSurviveTrophies` in the if statement, perhaps move it inside it